### PR TITLE
ut: ovn sb suite

### PR DESCRIPTION
**Pull request by zcq98**
# Pull Request

- [x] Make sure you have followed [Kube-OVN Code Style](https://github.com/kubeovn/kube-ovn/blob/master/CODE_STYLE.md).

## What type of this PR

Examples of user facing changes:

- Tests

## Which issue(s) this PR fixes

Fixes #(issue-number)

### CODE REVIEW

**Comment by zcq98** [comment]
@bobz965 Could you help review this?

File: pkg/controller/controller.go [hunk 1]
```diff
@@ -519,14 +519,17 @@ func Run(ctx context.Context, config *Configuration) {
 		config.OvnNbAddr,
 		config.OvnTimeout,
 		config.OvsDbConnectTimeout,
-		config.OvsDbInactivityTimeout); err != nil {
+		config.OvsDbInactivityTimeout,
+		60,
```
Comment (bobz965) [inline pkg/controller/controller.go:523]: 60 --> ovsDbInactivityTimeout, use a const like above

File: pkg/ovs/ovn-nb-acl_test.go [hunk 2]
```diff
@@ -68,7 +68,7 @@ func (suite *OvnClientTestSuite) testUpdateIngressACLOps() {
 	t := suite.T()
 	t.Parallel()
 
-	ovnClient := suite.ovnClient
+	ovnClient := suite.ovnNBClient
```
Comment (bobz965) [inline pkg/ovs/ovn-nb-acl_test.go:71]: nbClient := suite.ovnNBClient is better

File: pkg/ovs/ovn-nb-suite_test.go [hunk 3]
```diff
@@ -904,3 +962,55 @@ func newNbClient(addr string, timeout int) (client.Client, error) {
...
+		ovsDbClient: ovsDbClient{
+			Client:  nbClient,
+			Timeout: time.Duration(ovnSbTimeout) * time.Second,
+		},
+	}, nil
+}
+
+func newSbClient(addr string, timeout int) (client.Client, error) {
+	dbModel, err := ovnsb.FullDatabaseModel()
+	if err != nil {
```
Comment (bobz965) [inline pkg/ovs/ovn-nb-suite_test.go:980]: use require.NoError(t, err) like above

File: pkg/ovs/ovn-nb-suite_test.go [hunk 4]
```diff
@@ -904,3 +962,55 @@ func newNbClient(addr string, timeout int) (client.Client, error) {
...
+		client.WithLeaderOnly(false),
+		client.WithLogger(&logger),
+	}
+
+	for _, ep := range strings.Split(addr, ",") {
+		options = append(options, client.WithEndpoint(ep))
+	}
+
+	c, err := client.NewOVSDBClient(dbModel, options...)
+	if err != nil {
```
Comment (bobz965) [inline pkg/ovs/ovn-nb-suite_test.go:1000]: use require.NoError(t, err)

File: pkg/ovs/ovn-sb-chassis_test.go [hunk 5]
```diff
@@ -0,0 +1,548 @@
...
+
+	chassis := newChassis(0, "host-name-1", "chassis-name-1", nil, nil, nil, nil, nil)
+	ops, err := ovnClient.ovsDbClient.Create(chassis)
+	require.NoError(t, err)
+	err = ovnClient.Transact("chassis-add", ops)
+	require.NoError(t, err)
+
+	t.Run("test get chassis", func(t *testing.T) {
+		chassis, err := ovnClient.GetChassis("chassis-name-1", false)
+		require.NotNil(t, chassis)
```
Comment (bobz965) [inline pkg/ovs/ovn-sb-chassis_test.go:46]: 先判断err 后判断资源

File: pkg/ovs/ovn-sb-chassis_test.go [hunk 6]
```diff
@@ -0,0 +1,548 @@
...
+		require.Nil(t, chassis)
+		require.ErrorContains(t, err, "failed to get chassis")
+	})
+}
+
+func (suite *OvnClientTestSuite) testDeleteChassis() {
+	t := suite.T()
+	t.Parallel()
+
+	ovnClient := suite.ovnSBClient
```
Comment (bobz965) [inline pkg/ovs/ovn-sb-chassis_test.go:73]: sbClient := suite.ovnSBClient

File: pkg/ovs/ovn-sb-chassis_test.go [hunk 7]
```diff
@@ -0,0 +1,548 @@
...
+		err := ovnClient.DeleteChassis("chassis-non-existent")
+		require.NoError(t, err)
+	})
+}
+
+func (suite *OvnClientTestSuite) testUpdateChassis() {
+	t := suite.T()
+	t.Parallel()
+
+	ovnClient := suite.ovnSBClient
```
Comment (bobz965) [inline pkg/ovs/ovn-sb-chassis_test.go:114]: sbclient 全局搜索重命名下

File: pkg/ovs/ovn-nb-load_balancer_health_check_test.go [hunk 8]
```diff
@@ -17,7 +17,7 @@ func (suite *OvnClientTestSuite) testAddLoadBalancerHealthCheck() {
 	t.Parallel()
 
 	var (
-		ovnClient        = suite.ovnClient
+		ovnClient        = suite.ovnNBClient
```
Comment (bobz965) [inline pkg/ovs/ovn-nb-load_balancer_health_check_test.go:20]:
```suggestion
		nbClient        = suite.ovnNBClient
```

Comment (bobz965) [reply pkg/ovs/ovn-nb-load_balancer_health_check_test.go:20]: 这个文件全局替换下

File: pkg/ovs/ovn-sb-chassis_test.go [hunk 9]
```diff
@@ -0,0 +1,552 @@
...
+)
+
+func newChassis(nbcfg int, hostname, name string, encaps, transportZones, vtepLogicalSwitches []string, externalIDs, otherConfig map[string]string) *ovnsb.Chassis {
+	return &ovnsb.Chassis{
+		UUID:                ovsclient.NamedUUID(),
+		Encaps:              encaps,
+		ExternalIDs:         externalIDs,
+		Hostname:            hostname,
+		Name:                name,
+		NbCfg:               nbcfg,
```
Comment (bobz965) [inline pkg/ovs/ovn-sb-chassis_test.go:20]: 这里是 nbcfg 还是 sbcfg？

File: pkg/ovs/ovn-nb-load_balancer_health_check_test.go (hunk 8 above)
Comment (bobz965) [reply pkg/ovs/ovn-nb-load_balancer_health_check_test.go:20]: 感觉这个pr 已经太大了，下次再单独提吧

**Review by bobz965 — APPROVED** [review]

**Comment by bobz965** [comment]
@oilbeater 大佬，这个帮忙review下，大部分都是明明替换，新增了一个 sb 的测试

File: pkg/ovs/ovn-sb-chassis_test.go (hunk 9 above)
Comment (zcq98) [reply pkg/ovs/ovn-sb-chassis_test.go:20]: image
应该是nbcfg

**Review by bobz965 — APPROVED** [review]

**Review by oilbeater — APPROVED** [review]

**Pull request merged**